docs/admin/cluster-management.md

```diff
@@ -169,8 +169,8 @@ This is an infrequent event, but it requires careful management. There is a sequ
    1. Update existing objects in the storage to new version by running `cluster/update-storage-objects.sh`.
    1. Turn off the old API version.
 
+<a name="Turn-on-or-off-an-api-version-for-your-cluster"></a>
```

Comment:
All - thanks for the guidance. I made the relevant changes and I'd like to resubmit this PR.